benchflow-test-manager/application/src/test/java/cloud/benchflow/testmanager/TestManagerSmokeTestsIT.java

```diff
@@ -0,0 +1,191 @@
+  @Test
+  public void runLoadTest() throws Exception {
+
+    String testName = "WfMSLoadTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getLoadTestBundleFile(temporaryFolder), MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+  }
+
+  @Test
+  public void runExplorationExhaustiveOneAtATimeUsersTest() throws Exception {
+
+    String testName = "WfMSExhaustiveExplorationOneAtATimeSelectionUsersTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestExplorationOneAtATimeUsersBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+  }
+
+  @Test
+  public void runExplorationExhaustiveOneAtATimeMemoryTest() throws Exception {
+
+    String testName = "WfMSExhaustiveExplorationOneAtATimeSelectionMemoryTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestExplorationOneAtATimeMemoryBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+  }
+
+  @Test
+  public void runTestExplorationRandomUsersTest() throws Exception {
+
+    String testName = "WfMSExhaustiveExplorationRandomSelectionUsersTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestExplorationRandomUsersBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+  }
+
+  @Test
+  public void runTestExplorationExhaustiveOneAtATimeUsersEnvironmentTest() throws Exception {
+
+    String testName = "WfMSExhaustiveExplorationOneAtATimeSelectionUsersEnvironmentTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestExplorationOneAtATimeUsersEnvironmentBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+
+  }
+
+  @Test
+  public void runTestTerminationCriteria() throws Exception {
+
+    String testName = "TestTerminationCriteriaTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestTerminationCriteriaBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+
+  }
+
+  @Test
+  public void runTestStepUsers() throws Exception {
+
+    String testName = "WfMSStepUsersExhaustiveExplorationTest";
+
+    FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("benchFlowTestBundle",
+        TestBundle.getTestStepUsersBundleFile(temporaryFolder),
+        MediaType.APPLICATION_OCTET_STREAM_TYPE);
+
+    runTest(testName, fileDataBodyPart);
+
+
+  }
+
+  private void runTest(String testName, FileDataBodyPart fileDataBodyPart) {
```

Comment:
They are executed, but we do not care about it, because we just check that the API respond as expected. `TestTaskSchedulerIT` tests for correct execution of tests.